library/src/main/java/com/getstream/sdk/chat/view/messageinput/MessageInputView.kt

```diff
@@ -356,12 +355,6 @@ class MessageInputView(context: Context, attrs: AttributeSet?) : RelativeLayout(
         gridLayoutManager.spanCount = MEDIA_ITEMS_PER_ROW
     }
 
-    internal fun showTotalFileAttachments(totalFileAttachmentAdapter: FileAttachmentListAdapter) {
-        gridSpacingItemDecoration.setSpanCount(FILE_ITEMS_PER_ROW)
```

Comment:
Do we still need the constants? (MEDIA_ITEMS_PER_ROW, FILE_ITEMS_PER_ROW) Maybe just use them in init...

Reply:
Nice one! Thanks!